src/components/DrinkDetails/__tests__/DrinkDetails.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import DrinkDetails from '../DrinkDetails';
+
+const match = {
+  params: {
+    id: 11002,
+  },
+};
+
+const wrapper = shallow(<DrinkDetails match={match} />);
+
+it('should mount', () => {
+  expect(wrapper).toBeDefined();
+});
+
+// it('should have 5 ingredients', () => {
```

Comment:
Why is this test commented out?

Reply:
I was going to test for a specific drink but realized I would need to fetch the data (or mock it) and I hadn't learned doing the async or mocking in a test yet.